src/state/selector-utils.ts

```diff
@@ -1,7 +1,7 @@
 import isEqual from 'lodash/isEqual'
-import { createSelectorCreator, defaultMemoize } from 'reselect'
+import { createSelectorCreator, lruMemoize } from 'reselect'
```

Comment:
See https://redux.js.org/usage/migrations/migrating-rtk-2#defaultmemoize-renamed-to-lrumemoize